tests/Unit/Domain.LinnApps.Tests/ImportBookServiceTests/WhenUpdating.cs

```diff
@@ -0,0 +1,214 @@
+                FlightNumber = string.Empty,
+                TransportId = 1,
+                TransportBillNumber = string.Empty,
+                TransactionId = 44,
+                DeliveryTermCode = string.Empty,
+                ArrivalPort = "LAX",
+                LineVatTotal = null,
+                Hwb = "what is hwb",
+                SupplierCostCurrency = "bacon",
+                TransNature = "air",
+                ArrivalDate = null,
+                FreightCharges = null,
+                HandlingCharge = null,
+                ClearanceCharge = null,
+                Cartage = null,
+                Duty = null,
+                Vat = null,
+                Misc = null,
+                CarriersInvTotal = null,
+                CarriersVatTotal = null,
+                TotalImportValue = (decimal)123.4,
+                Pieces = null,
+                Weight = null,
+                CustomsEntryCode = "code RED",
+                CustomsEntryCodeDate = null,
+                LinnDuty = null,
+                LinnVat = null,
+                IprCpcNumber = null,
+                EecgNumber = null,
+                DateCancelled = null,
+                CancelledBy = null,
+                CancelledReason = null,
+                CarrierInvNumber = null,
+                CarrierInvDate = null,
+                CountryOfOrigin = string.Empty,
+                FcName = string.Empty,
+                VaxRef = string.Empty,
+                Storage = null,
+                NumCartons = null,
+                NumPallets = null,
+                Comments = string.Empty,
+                ExchangeRate = null,
+                ExchangeCurrency = string.Empty,
+                BaseCurrency = string.Empty,
+                PeriodNumber = null,
+                CreatedBy = null,
+                PortCode = string.Empty,
+                CustomsEntryCodePrefix = "AA",
+                InvoiceDetails = new List<ImportBookInvoiceDetail>(),
+                OrderDetails = new List<ImportBookOrderDetail>(),
+                PostEntries = new List<ImportBookPostEntry>()
+            };
+
+            this.newImpBook = new ImportBook
+            {
+                Id = this.impbookId,
+                DateCreated = now.AddDays(2),
+                ParcelNumber = 1,
+                SupplierId = 556,
+                ForeignCurrency = "YN",
+                Currency = "GBD",
+                CarrierId = 678,
+                OldArrivalPort = "Glesga",
+                FlightNumber = "sk123",
+                TransportId = 2,
+                TransportBillNumber = "1212",
+                TransactionId = 45,
+                DeliveryTermCode = "dli",
+                ArrivalPort = "LAZ",
+                LineVatTotal = 12,
+                Hwb = "hwbbb",
+                SupplierCostCurrency = "egg",
+                TransNature = "sea",
+                ArrivalDate = this.now.AddDays(3),
+                FreightCharges = (decimal)11.1,
```

Comment:
you can do 11.1m to specify decimal which is a bit neater I find

Reply:
+1